pkg/services/tokenexchanger/setting_access_token_signing.go

```diff
@@ -0,0 +1,46 @@
+package tokenexchanger
+
+import (
+	authlib "github.com/grafana/authlib/authn"
+	sdkhttpclient "github.com/grafana/grafana-plugin-sdk-go/backend/httpclient"
+	"github.com/grafana/grafana/pkg/setting"
+)
+
+const (
+	configSection  = "access_token_signing"
```

Comment:
We also have the `grpc_client_authentication` section with the same attributes:
```
[grpc_client_authentication]
token=
token_exchange_url=
token_namespace=
```
It might be worth renaming the section and use only one.

Reply:
https://github.com/grafana/grafana/blob/2dee9ccbbcd0a01c2f0afcc89c5e0f5f03aa602a/pkg/services/authn/grpcutils/config.go#L61

Reply:
We should most likely reuse the same exchanger in all places in st grafana

Reply:
i can try refactoring this usage to use the same injected exchanger, as the one introduced by this PR.

Reply:
Renaming the section feels like the right choice. Are you happy with the new section name i picked in this PR?

Reply:
I actually got confused and made the scope of this PR more than it needs to be.

From the new description:

>I will absorb the comments about the renaming the current grpc_authenticator section's key for the CAP token in a different PR focused on single-tenant Grafana.

This PR is for unistor server. I will situate the keys under the existing unistor section.